### ISSUE TITLE
Integrate Hyperledger Interface in AEGIS

### ISSUE THREAD

**Issue by FabiApfelkern** [aegisbigdata/documentation#16]

**Comment by skous** [comment]
Which interface do you mean? 
I think it makes sense to showcase the latest transactions history for each user.
But other kind of interfaces, I don't know.

**Comment by FabiApfelkern** [comment]
Yes I only mean to display relevant information coming from the Hyperledger REST API
